app/routes/healthy.js

```diff
@@ -2,6 +2,7 @@ const route = {
   method: 'GET',
   path: '/healthy',
   handler: (request, h) => {
+    request.logger.info('Healthy check')
```

Comment:
Question for my own curiosity - does this just log that the route is healthy rather than needing to go to the `/healthy` endpoint itself?

Reply:
No idea :D Not touched this file as part of implementation, since it is out of scope and the fact that they come as part of initial setup :)

Reply:
That line appears as a newly added line on this PR so I thought it might have been for local development.

Reply:
@simonnineteen90 ?

Reply:
@JungaBunga GitLens is saying you added this line 😆. Not an issue though - I was just being curious really.

Reply:
this was when we were testing using the Hapi Pino plugin for debugging the Graphql/Apollo plugins.

It can be changed back to `console.log`

Reply:
@simonnineteen90 Ahh cool, thanks for the explanation! 🙌🏽